plant-genetic-resources-characterization-and-utilization.csl

```diff
@@ -0,0 +1,346 @@
+          <text value="See"/>
+          <text variable="URL" font-style="normal"/>
+          <group prefix="(" suffix=")">
+            <text term="accessed" suffix=" "/>
+            <date variable="accessed">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+      <else-if type="article-magazine article-newspaper" match="any">
+        <date variable="issued" prefix=" ">
+          <date-part name="day" suffix=" "/>
+          <date-part name="month"/>
+        </date>
+      </else-if>
+      <else-if type="report"/>
+      <else-if type="article-journal bill book graphic legal_case legislation motion_picture song" match="any">
+        <date variable="issued">
+          <date-part name="day" prefix=" "/>
+          <date-part name="month" prefix=" "/>
+        </date>
+      </else-if>
+      <else>
+        <group prefix=" " delimiter=" ">
+          <group prefix="(" suffix=")">
+            <text term="accessed" suffix=" "/>
+            <date variable="accessed">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <choose>
+          <if type="webpage">
+            <choose>
+              <if variable="accessed">
+                <date variable="accessed">
+                  <date-part name="year"/>
+                </date>
+              </if>
+              <else>
+                <text term="no date" form="short"/>
+              </else>
+            </choose>
+          </if>
+          <else>
+            <text term="no date" form="short"/>
+          </else>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="extra">
+    <choose>
+      <if variable="note">
+        <text variable="note" prefix=". "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="un-sales">
+    <choose>
+      <if variable="call-number">
+        <text variable="call-number" prefix="United Nations publication. Sales No. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal">
+        <group delimiter=": ">
+          <text variable="volume"/>
+          <text variable="page"/>
+        </group>
+      </if>
+      <else-if type="report">
+        <text variable="number" prefix=" No. "/>
+      </else-if>
+      <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference song" match="any">
+        <choose>
+          <if variable="collection-title">
+            <group delimiter=", ">
```

Comment:
handle the spaces here more elegantly (e.g. no need for the one before "No." and the one before "Vol." should probably be a group prefix)